src/main/resources/assets/alaskanativecraft/lang/de_de.json

```diff
@@ -0,0 +1,68 @@
+{
+  "_credit": "Translations provided by koelle25",
+  "itemGroup.alaskanativecraft.items": "Alaska Native Craft",
+  "item.alaskanativecraft.muktuk": "Maktaaq",
+  "item.alaskanativecraft.seal": "Rohes Robbenfleisch",
+  "item.alaskanativecraft.cooked_seal": "Gebratenes Robbenfleisch",
+  "item.alaskanativecraft.ptarmigan": "Rohes Schneehuhnfleisch",
+  "item.alaskanativecraft.cooked_ptarmigan": "Gebratenes Schneehuhnfleisch",
+  "item.alaskanativecraft.venison": "Rohes Wildfleisch",
+  "item.alaskanativecraft.cooked_venison": "Gebratenes Wildfleisch",
+  "item.alaskanativecraft.salmonberries": "Lachsbeeren",
+  "item.alaskanativecraft.raspberries": "Himbeeren",
+  "item.alaskanativecraft.cloudberries": "Moltebeeren",
+  "item.alaskanativecraft.blueberries": "Heidelbeeren",
+  "item.alaskanativecraft.akutaq": "Akutaq",
+  "item.alaskanativecraft.ulu": "Ulu",
+  "item.alaskanativecraft.wooden_harpoon": "Holzharpune",
+  "item.alaskanativecraft.stone_harpoon": "Steinharpune",
+  "item.alaskanativecraft.iron_harpoon": "Eisenharpune",
+  "item.alaskanativecraft.golden_harpoon": "Goldharpune",
+  "item.alaskanativecraft.diamond_harpoon": "Diamantharpune",
+  "item.alaskanativecraft.netherite_harpoon": "Netheritharpune",
+  "item.alaskanativecraft.kuspuk_hood": "Kuspuk-Kaputze",
+  "item.alaskanativecraft.kuspuk_body": "Kuspuk",
+  "item.alaskanativecraft.snow_goggles": "Schneebrille",
+  "item.alaskanativecraft.kuspuk_skirt": "Kuspuk-Rock",
```

Comment:
I actually forgot that this is unused, so you can remove it if you'd like

Reply:
In-Game (thanks to REI) I can see that `kuspuk_hood` and `kuspuk_body` are valid items with existing crafting recipes. `snow_goggles` are also there, but have no recipe. Only `kuspuk_skirt` seems to be completely missing/unused. Would it be best to then only remove the `kuspuk_skirt` translation line?